scripts/import_osm.py

```diff
@@ -0,0 +1,111 @@
+import argparse
+import sys
+import osmnx as ox
+from ipygis import get_connection_url
+from sqlalchemy import create_engine
+from sqlalchemy.engine.base import Engine
+from sqlalchemy.schema import CreateSchema
+from geoalchemy2 import Geometry, WKTElement
+from typing import Dict
+
+# test simple import now, convert to module later
+sys.path.insert(0, "..")
+from models import OSMPoint
+from osm_tags import tags_to_filter
+
+
+class OsmImporter(object):
+    """Import OSM data to PostGIS using Overpass API."""
+
+    def __init__(self, args: Dict):
+        args = self._parse_args(args)
+        self.slug = args["slug"]
+        self.bbox = args["bbox"]
+        self.city = args["city_name"]
+        self._engine = self._create_engine()
+
+    @staticmethod
+    def _parse_args(args) -> Dict:
+        """Parse input arguments to a dict."""
+        slug = args["city"]
```

Comment:
Ah, one more thing. In the database, the slug should always be lowercase, so I think we should use slugify here, or perhaps add argument called `slug` like in the other importers.

Actually `slug` argument is probably needed, because import.py calls also other importers by slug. This way, all the imported datasets will end up in the same schema with lower-case slug.

Reply:
Actually misread this. This slug only means the old-fashioned three-letter slug we don't need. Just get rid of three-letter slugs.